tests/Transports.Subscriptions.WebSockets.Tests/TestStartup.cs

```diff
@@ -10,6 +11,12 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<TestSchema>();
             services.AddGraphQL()
                 .AddWebSockets();
+            services.AddLogging(builder =>
+            {
+                // prevent writing errors to Console.Error during tests (required for testing on ubuntu)
```

Comment:
Wow! Does this cause the tests to fail?

Reply:
What about allow to write logs on Windows?

```c#
if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
{
  // Do something
}
```

Reply:
I can’t remember anymore. Maybe it just caused a bunch of extra output. I think it caused a failure though, without this change. I did write “required”...

Reply:
Well, OK.